packages/pagination/src/styled/OffsetPagination/StyledPageBase.ts

```diff
@@ -69,6 +69,12 @@ const colorStyles = ({ theme }: ThemeProps<DefaultTheme>) => {
       inset: true
     })}
 
+    &:disabled,
+    [aria-disabled='true'] {
+      background-color: transparent;
+      color: ${disabledColor};
+    }
```

Comment:
Why was this moved? The result is that `:active` styling takes precedent over `:disabled`, which is not correct.